doc/manual/source/data/updating.rst

```diff
@@ -17,32 +17,32 @@ Combining different feature types
 
-Imagine that you already imported a shapefile containing polygons with a given set of attributes, and now you want to import into the same path another shapefile with polygons, which contain the same attributes but with an extra one containing the area of each polygon. In that case, feature types do not match, and the situation is different, but GeoGig provide tools to solve it. 
+Imagine that you already imported a shapefile containing polygons with a given set of attributes, and now you want to import into the same path another shapefile with polygons, which contain the same attributes but with an extra one containing the area of each polygon. In that case, feature types do not match, and the situation is different, but GeoGig provide tools to solve it.
 
 By default, GeoGig will not let you import into an existing tree if the feature type of the data to import is not the same one as the feature type of the existing tree. This is to prevent trees with mixed feature types to appear, so in case you want that to happen, you have to explicitly tell GeoGig to do it.
 
 If the feature type is different, GeoGig will try to match it to the default existing feature type before aborting the import operation. This is done to allow using different data formats to be used seamlessly, since they might store a given feature type differently. Let's see an example.
 
-Let's say that you have imported a shapefile with an attribute called ``Area``. Your feature type will be like this
+Let's say that you have imported a shapefile with an attribute called ``Area``. The feature type will appear as follows.
 
 ``the_geom``: MULTIPOLYGON
 ``Area``: DOUBLE
 
-Imagine that you export it to a GeoJSON file, make some changes and then reimport it. The feature type of the data to import is like this.
+Imagine that you export it to a GeoJSON file, make some changes and then re-import it. The feature type of the data to import is the following.
 
 ``Area`` : DOUBLE
 ``geom`` : MULTIPOLYGON
 
-Although the feature type is the same, the GeoTools library used by GeoGig understands the file to import in a different way, putting the geometry field as the last one instead of the first one, and using a different name. GeoGig will, however, understand that the feature type is the same one as the existing one, and will correctly set that existing feature type as the feature type of the imported features.
+Although the feature type is the same, the GeoTools library used by GeoGig understands the file to import in a different manner, putting the geometry field as last instead of first, and using a different name. GeoGig will, however, understand that the feature type is the same as the existing feature and will correctly set that existing feature type as the feature type of the imported feature.
```

Comment:
This sentence:

GeoGig will, however, understand that the feature type is the same **one** as the existing **one**, and will correctly set that existing feature type as the feature type of the imported **features**.

shouldn't have the second **one** replaced with **feature**, it refers to **feature type**.

Also, the **features** at the end is *more correct* as again, it's referring to the feature type of the imported features. Perhaps a reword of this sentence should be:

`GeoGig will, however, understand that the feature type is the same as the existing feature type, and will correctly set that existing feature type as the feature type of the imported features.`